h-w2

```diff
@@ -0,0 +1,86 @@
+#1
+#1.1
+
+data = [
+    {'name': 'Viktor', 'city': 'Kiev', 'age': 30 },
+    {'name': 'Maksim', 'city': 'Dnepr', 'age': 20},
+    {'name': 'Vladimir', 'city': 'Lviv', 'age': 32},
+    {'name': 'Andrey', 'city': 'Kiev', 'age': 34},
+    {'name': 'Artem', 'city': 'Dnepr', 'age': 50},
+    {'name': 'Dmitriy', 'city': 'Lviv', 'age': 21}]
+data.sort(key=lambda d: d['age'])
+print (t)
```

Comment:
`print()` should be removed or added variable `t`, because for now, variable `t` is undefined.